### PR TITLE
Iz style ws view

### PR DESCRIPTION
**Pull request by zarsky-broad**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -4,10 +4,11 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="stylesheet" type="text/css" href="https://cloud.typography.com/7761236/7617792/css/fonts.css" />
     <script src="https://apis.google.com/js/platform.js"></script>
     <style>
       * {
-        font-family: Arial, Helvetica, sans-serif;
+        font-family: "Gotham SSm A", "Gotham SSm B", sans-serif;
```
Comment (dmohs) [inline public/index.html:11]: Sure you don't want a fallback?

File: public/index.html [hunk 2]
```diff
@@ -4,10 +4,11 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="stylesheet" type="text/css" href="https://cloud.typography.com/7761236/7617792/css/fonts.css" />
```
Comment (dmohs) [inline public/index.html:7]: This probably needs to be configurable. We can depend on their servers for dev, but we probably want to serve the production version from our own servers. No need to fix in this PR, but we've said that a few times now, so let's start actually filing tickets when we do that.

File: src/components/common.js [hunk 3]
```diff
@@ -29,7 +29,7 @@ const card = function(props, children) {
       style: {
         borderRadius: 5, padding: '1rem', wordWrap: 'break-word',
         backgroundColor: 'white',
-        boxShadow: '0 0 2px 0 rgba(0,0,0,0.12), 0 3px 2px 0 rgba(0,0,0,0.12)'
+        boxShadow: Style.standardShadow
```
Comment (dmohs) [inline src/components/common.js:32]: I love that you are conscientious about doing this kind of thing.

File: src/components/icons.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+     xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true" role="img">
+    <style type="text/css">line {
+      stroke: currentColor;
+      stroke-width: 2;
+    }</style>
+    <path d="M15,2l0,13L2,15L2,2H15 M15,0H2C0.9,0,0,0.9,0,2v13c0,1.1,0.9,2,2,2h13c1.1,0,2-0.9,2-2V2C17,0.9,16.1,0,15,0L15,0z"/>
+    <line x1="8.5" y1="17" x2="8.5" y2="0"/>
+    <line x1="0" y1="6" x2="17" y2="6"/>
+    <line x1="0" y1="11" x2="17" y2="11"/>
+</svg>`
```
Comment (dmohs) [inline src/components/icons.js:29]: Tell me more about this tomorrow. I'm just interested.

**Review by dmohs — APPROVED** [review]

File: public/index.html (hunk 1 above)
Comment (zarsky-broad) [reply public/index.html:11]: TBH, sans-serif is probably fine, whatever we'd define would likely be the default for that (Arial, Helvetica)

File: public/index.html (hunk 2 above)
Comment (zarsky-broad) [reply public/index.html:7]: Yes, they very specifically recommend that you serve it yourself in production. They recommend we shift when we're getting ready for people to actually use the app: https://dashboard.typography.com/user-guide/moving-to-production

File: src/components/common.js (hunk 3 above)
Comment (zarsky-broad) [reply src/components/common.js:32]: Meh, it gets used in multiple places, why repeat myself?

File: src/components/icons.js (hunk 4 above)
Comment (zarsky-broad) [reply src/components/icons.js:29]: Haha, there is an icon in the screens that isn't in Clarity, so I recreated it in Illustrator, exported it as SVG, and trimmed the _frack_ out of the resultant code until just the relevant parts remained, using Clarity's own SVG code as a template for what actually matters.

File: public/index.html (hunk 1 above)
Comment (zarsky-broad) [reply public/index.html:11]: (But I'll ask Lou when we're back in the office)

**Pull request merged**